### ISSUE TITLE
Pwned Password API Support? 

### ISSUE THREAD

**Issue by Grantimus9** [techgaun/ex_pwned#2]
Hello, 

Do you have any intent to add support for checking if a password was in a breach using HIPB's new password checker API? 

I think it'd add some great functionality, and would be useful for the Elixir community. I might see if I can work up a PR.

**Comment by techgaun** [comment]
@Grantimus9 would be great if you can make a PR. Otherwise, I can knock out this one over the weekend. Feel free to take this issue if you wanna work on it.

**Comment by Grantimus9** [comment]
I'll take a stab at it. 

The simple endpoint is being retired, so we'll be using the "range" one: https://haveibeenpwned.com/API/v2#PwnedPasswords

How about: 
```
# returns true or false 
ExPwned.password_breached?(password) 

# Returns number of times this password appears in a breach. Zero +  
ExPwned.password_breached_count(password) 
```

**Comment by techgaun** [comment]
@Grantimus9 yep that sounds good to me :+1:

**Comment by techgaun** [comment]
closed by #3
